### DISCUSSION TITLE
Query template most used.

### DISCUSSION THREAD

**Discussion by adumith** [microsoft-search/pnp-modern-search#3883] [Q&A] [answered]
Hello everyone, 

I know that this may vary according to the needs of each one, but I was wondering if you could share the most frequent parameters to make the query, for example in my case I have these:

{inputQueryText}
“SPS-HideFromAddressLists”=0
owstaxIdSPShDepartment: “#8ed8c9ea-7052-4c1d-a4d7-b9c10bffea6f”

Basically here I am showing only the user accounts that are active and those whose departments are not blank.

Any others that might be useful?

Thank you very much,

**Comment by kasperbolarsen** [comment, marked as answer]
I have a number of blog posts about this, like this one https://www.m365thinking.com/post/sharepoint-people-search-how-to-clean-up-your-results-by-excluding-accounts

**Reply by adumith** [reply]
Excellent, thank you very much Kasper.

I already have my filter. Now I am still dealing with the department problem, for which I have posted another thread.

Thank you very much sir.
